docs/reference/aggregations/metrics/boxplot-aggregation.asciidoc

```diff
@@ -166,6 +166,35 @@ GET latency/_search
 
 include::percentile-aggregation.asciidoc[tags=t-digest]
 
+==== Accuracy
+
+The default implementation of TDigest is optimized for performance, scaling to millions or even
+billions of sample values while maintaining acceptable accuracy levels (close to 1% relative error
+for millions of samples in some cases). There's an option to use an implementation optimized
+for accuracy that can be selected through parameter `optimize_for_accuracy`:
+
+[source,console]
+--------------------------------------------------
+GET latency/_search
+{
+  "size": 0,
+  "aggs": {
+    "load_time_boxplot": {
+      "boxplot": {
+        "field": "load_time",
+        "optimize_for_accuracy": true    <1>
```

Comment:
I think a boolean is too limiting here. If we want to add another option later, we'd have to change this parameter, and the client's team considers it breaking to move from a boolean to an enum.  I like the way we did this for Terms.  There, we added an `execution_hint` parameter, which we can add values to later if we need to.  See https://www.elastic.co/guide/en/elasticsearch/reference/current/search-aggregations-bucket-terms-aggregation.html#search-aggregations-bucket-terms-aggregation-execution-hint

Reply:
in case it wasn't clear, I'm specifically saying the parameter should be called `execution_hint` for consistency

Reply:
Sounds good, I added `execution_hint` with value `high_accuracy` instead.